pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -38,10 +38,10 @@ func NewCertRotationController(
 			EventRecorder:     eventRecorder,
 		},
 		certrotation.CABundleRotation{
-			Namespace:     operatorclient.GlobalMachineSpecifiedConfigNamespace,
-			Name:          "aggregator-client-ca",
-			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.GlobalMachineSpecifiedConfigNamespace).Core().V1().ConfigMaps(),
-			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.GlobalMachineSpecifiedConfigNamespace).Core().V1().ConfigMaps().Lister(),
+			Namespace:     operatorclient.OperatorNamespace,
```

Comment:
this is *part* of the complete bundle.  Since it is under management by this operator only, it should live in the operator namespace